src/components/BlogPage/BlogPageTemplate.js

```diff
@@ -21,11 +21,13 @@ const styles = theme => ({
 	header: {
 		// align icon with text
 		display: 'flex',
-		alignItems: 'center'
+		alignItems: 'center',
+		justifyContent: 'center',
+		marginTop: theme.spacing(8)
```

Comment:
rly rly rly rly tiny nit: the dist from the top of the page to the header is different on the blog post page than the other pages and idk if that's something you feel like fixing. It's kinda out of the scope of this PR so understandable if you don't want to